src/setup-ros-windows.ts

```diff
@@ -4,17 +4,54 @@ import * as chocolatey from "./package_manager/chocolatey";
 import * as pip from "./package_manager/pip";
 import * as utils from "./utils";
 
-const rosdepBin: string =
-	"c:\\hostedtoolcache\\windows\\python\\3.6.8\\x64\\scripts\\rosdep";
+const python37: string =
+	"C:\\hostedtoolcache\\windows\\Python\\3.7.6\\x64";
+
+const binaryReleases: { [index: string]: string; } =
+	{
+		"dashing": "https://github.com/ros2/ros2/releases/download/release-dashing-20191213/ros2-dashing-20191213-windows-amd64.zip",
+		"eloquent": "https://github.com/ros2/ros2/releases/download/release-eloquent-20200124/ros2-eloquent-20200124-windows-release-amd64.zip"
+	};
+
+const pip3Packages: string[] = [
+	"numpy",
+	"netifaces",
+	"lxml"
```

Comment:
Could you keep the array alphabetically sorted for consistency?

Reply:
Done.